go/vt/throttler/replication_lag_cache.go

```diff
@@ -76,9 +82,35 @@ func (c *replicationLagCache) add(r replicationLagRecord) {
 	entry.add(r)
 }
 
+// maxLag returns the maximum replication lag for the entries in cache.
+func (c *replicationLagCache) maxLag() (maxLag uint32) {
```

Comment:
Moved this mostly-unchanged from `throttler.go` instead of holding the `mu sync.Mutex` lock from a different struct